### PR TITLE
Made some proposed updates to basicGitCommands.

### PR DESCRIPTION
**Pull request by chasb96**

### CODE REVIEW

File: CommandLine/basicGitCommands.md [hunk 1]
```diff
@@ -26,41 +26,50 @@ If you need to know anything about git, it's these next four commands.
...
 ### Tagging Convention:
-- First Number (Major) **1**.0.0
+- First Number (Major) **x**.x.x
     - MAJOR changes, large overhauls, major portions of the application, etc.
-- Second Number (Minor) 1.**1**.0
-    - Minor features added. 
-- Third Number (Patch) 1.1.**1**
+    - Typically a release
+- Second Number (Minor) x.**x**.x
+    - Minor changes not worthy of creating a full release for.
```
Comment (melikai) [inline CommandLine/basicGitCommands.md]: Remove the word 'for'

File: CommandLine/basicGitCommands.md [hunk 2]
```diff
@@ -26,41 +26,50 @@ If you need to know anything about git, it's these next four commands.
...
 
 `git checkout <branch-name>`                        
-- Checks out a branch. Use `-b` to create a new branch
+- Checks out a branch. Use `-b` to create a new branch off of the current checked-out branch. (This will automatically `checkout` the new branch)
 - Use both `git fetch` and `git checkout` at the same time to grab a branch from the remote repository and start making changes.
-    - `git fetch && git checkout <branch-name>`
+    - `git fetch ; git checkout <branch-name>`
 
-`git push`                           
+`git push [<remote>] [<branch-name>]`
```
Comment (melikai) [inline CommandLine/basicGitCommands.md]: What does this word <remote> mean? I wanted to keep these notes extremely simple so this should be explained

Comment (chasb96) [inline CommandLine/basicGitCommands.md]: Usually in help pages or man-pages  they'll give a full 'overview' of a command. They'll put `<>` for entered parameter text and `[]` for optional parameters.
So for example, `git branch` works as `git branch` and there's the param `-D branch-name`. So they'll annotate this as `git branch [-D <branch-name>]`

Comment (chasb96) [inline CommandLine/basicGitCommands.md]: Remote refers to the location to  use. Typically you use just `origin`, but you can change or add or delete remote's to use.

Comment (melikai) [inline CommandLine/basicGitCommands.md]: I still don't know what that word means, which tells me that it'll be confusing to someone else too

Comment (chasb96) [inline CommandLine/basicGitCommands.md]: Okay, I'll drop it out.

**Pull request merged**